### PR TITLE
feature: allow reduce karma & add mafia members

### PR DESCRIPTION
**Pull request by wilkadrian**

### CODE REVIEW

**Comment by const-dominik** [comment]
excellent job, exactly what this bot needed. hope this will get approved

**Comment by wilkadrian** [comment]
> excellent job, exactly what this bot needed. hope this will get approved

@const-dominik ++

**Review by const-dominik — APPROVED** [review]
code looks awesome, author is clearly talented (you can say that he has a head like a shop in polish), imo those changes are innovative and necesarry, and should get approved asap

**Comment by wilkadrian** [comment]
@const-dominik thank you Dominik for fast feedback. As you said, changes needs to merged asap

@mmiszy are you here?

File: src/data/karma.ts [hunk 1]
```diff
@@ -68,6 +68,10 @@ export const getEmojiForKarmaValue = (value: number) => {
 export const getKarmaDescription = (value: number) =>
   `${Math.floor(value)} XP (lvl ${getKarmaLevel(value)}) ${getEmojiForKarmaValue(value)}`;
 
+const KARMA_MAFIA_MEMBERS = ['349836529616814091', '363038285020266496']; // please dont edit this line
```
Comment (Eghizio) [inline src/data/karma.ts]: XDDD

**Comment by wilkadrian** [comment]
🕵🏿‍♂️

**Pull request closed without merging**